### DISCUSSION TITLE
GoogleMap capture turns out black

### DISCUSSION THREAD

**Discussion by alexandrupele** [PatilShreyas/Capturable#165] [General]
Hey, 

GoogleMap has a built-in snapshot function but it sometimes crashes and I am trying to use Capturable instead. The problem is the map turns out black (see screenshot). I am 100% sure the map is loaded and visible on the view and the capture even has the Google logo, however instead of the map it just shows black. I've seen some older posts (5y+) of people getting the same result when they were trying to capture the screenshot themselves. Any idea why this happens and how to work around it? Thanks!

![map-capture](https://github.com/PatilShreyas/Capturable/assets/8121902/f317723b-bd85-40b3-b9aa-4c7bc8f084e7)

**Comment by PatilShreyas** [comment]
@alexandrupele can you try this solution?
https://stackoverflow.com/a/30411748/11326621

**Reply by alexandrupele** [reply]
Hey @PatilShreyas , thank you for the swift reply! There is no WebView on the screen, it's a native Compose screen hosting a GoogleMap view. If you have any other ideas or suggestions, please let me know!

**Reply by PatilShreyas** [reply]
Sorry my bad! I just tried to search about Map and seeing similar issues already reported for capturing snapshot from the Map view. Let me get back once I found solution on this
